node/pkg/watchers/evm/watcher.go

```diff
@@ -634,6 +634,7 @@ func (w *Watcher) Run(ctx context.Context) error {
 		}
 	}()
 
+	readiness.SetReady(w.readiness)
```

Comment:
Unrelated change?

Reply:
Unrelated but related. When it can take a long time to reach finality (such as Arbitrum in testnet), it takes a long time for the runable to be marked ready, which causes tilt initialization to hang. This lets initialization continue sooner.

Reply:
Ok, then please add a comment here explaining that.

Reply:
Added a comment.